README.md

```diff
@@ -1,4 +1,7 @@
-![Logo](logo.gif)
+![Logo](https://raw.githubusercontent.com/idealista/node_exporter_role/master/logo.gif)
+
+[![Build Status](https://travis-ci.org/idealista/node_exporter_role.png)](https://travis-ci.org/idealista/node_exporter_role)
```

Comment:
`prometheus_node_exporter_role`, isn't it?

Reply:
ups